### PR TITLE
Add superlance to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
Superlance is a set of event processors for supervisord

### CODE REVIEW

File: recipes/superlance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - supervisor
+
+test:
+  imports:
+    - superlance
+    - superlance.tests
+
+about:
+  home: http://github.com/supervisor/superlance
+  license: BSD-variant
```
Comment (sodre) [inline recipes/superlance/meta.yaml]: They are using a variant of ZPL, what would you like me to choose?

**Comment by sodre** [comment]
@ocefpaf can you review this PR when you get a chance it is ready for review.

File: recipes/superlance/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/superlance/meta.yaml]: What is the text they are using to define? If they say BSD-variant or ZPL-variant, let us use that. If not just use 'superlance custom'.

**Comment by ocefpaf** [comment]
Thanks @sodre!

**Pull request merged**